src/hardware/cpe-ubiquiti.md

```diff
@@ -0,0 +1,117 @@
+# CPE - Ubiquiti
+
+This document covers setup for the following devices. Other devices may be similar.
+
+- [Loco5AC](https://www.ui.com/airmax/nanostation-ac/)
+- [LBE‑5AC‑23](https://www.ui.com/airmax/litebeam-ac/)
+
+## What you will need
+
+The following information you will need to collect before beginning  the configuration.
```

Comment:
You will need to collect the following information, and why is there a double space.

Reply:
fixed